### ISSUE TITLE
Categories in the export/import process get lost

### ISSUE THREAD

**Issue by fmontes** [dotCMS/core#19195]
**Describe the bug**
When you export a site from: `Maintenance > Tools > Download Data/Assets` and import in another instance:

1. The categories with children ends up all in main level
2. The contentlets with categories field ends up with the category field empty.

**To Reproduce**
1. Create categories with child
2. Create a content type with a category field
3. Create contentlets and add categories
4. Export the data from `Maintenance > Tools > Download Data/Assets`
5. Import the data in a new instance
6. Check the categories and check the contentlets

**Expected behavior**
Categories child and contentlets categories should be the same as the source.

## Acceptance Criteria

1. Fix categories in export/import
2. Work in all the supported browsers (don't forget IE11 and iPad)
3. Multilanguage
4. Unit test

**Comment by erickgonzalez** [comment]
Seems that the issue is at the time of importing since in the **Tree.xml** the entries are there
```
<com.dotmarketing.beans.Tree>
    <parent>dd98af23-c4ac-472b-aed4-60d67a797469</parent>
    <child>14ab7eca-7470-4f01-9671-a61f535c9b3a</child>
    <relationType>child</relationType>
    <treeOrder>0</treeOrder>
  </com.dotmarketing.beans.Tree>
  <com.dotmarketing.beans.Tree>
    <parent>dd98af23-c4ac-472b-aed4-60d67a797469</parent>
    <child>4c32d127-4f97-4db8-a1c8-c8934cc2b034</child>
    <relationType>child</relationType>
    <treeOrder>0</treeOrder>
  </com.dotmarketing.beans.Tree>
  <com.dotmarketing.beans.Tree>
    <parent>dd98af23-c4ac-472b-aed4-60d67a797469</parent>
    <child>8ec26ce5-0329-4f1e-9c55-30e6b8d04c87</child>
    <relationType>child</relationType>
    <treeOrder>0</treeOrder>
  </com.dotmarketing.beans.Tree>
  <com.dotmarketing.beans.Tree>
    <parent>dd98af23-c4ac-472b-aed4-60d67a797469</parent>
    <child>c1075aae-5392-4272-94ae-1c3ae01a6d86</child>
    <relationType>child</relationType>
    <treeOrder>0</treeOrder>
  </com.dotmarketing.beans.Tree>
```

**Comment by erickgonzalez** [comment]
PR: https://github.com/dotCMS/core/pull/19196

**Comment by fmontes** [comment]
We just need to assert that the categories field is set in the content type when you start dotcms with the full starter.

**Comment by bryanboza** [comment]
Fixed, tested on release-5.3.8 // Postgres // FF
